setup.py

```diff
@@ -23,18 +23,23 @@
       package_data={'wes_service': ['openapi/workflow_execution_service.swagger.yaml']},
       include_package_data=True,
       install_requires=[
-          'connexion',
-          'bravado',
+          'future',
+          'pytest',
```

Comment:
Install shouldn't require any packages that are used just for testing.

Reply:
Changed.